src/commander.c

```diff
@@ -881,12 +884,14 @@ static void commander_echo_2fa(char *command)
 static int commander_verify_signing(const char *message)
 {
     int data_len, type_len, keypath_len, change_keypath_len;
-    char *data, *type, *keypath, *change_keypath;
-
-    type = (char *)jsmn_get_value_string(message, CMD_STR[CMD_type_], &type_len);
-    data = (char *)jsmn_get_value_string(message, CMD_STR[CMD_data_], &data_len);
-    keypath = (char *)jsmn_get_value_string(message, CMD_STR[CMD_keypath_], &keypath_len);
-    change_keypath = (char *)jsmn_get_value_string(message, CMD_STR[CMD_change_keypath_],
+    const char *data, *type, *keypath, *change_keypath;
+
+    type = (const char *)jsmn_get_value_string(message, CMD_STR[CMD_type_], &type_len);
```

Comment:
`jsmn_get_value_string` returns a `const char *`, so no need for a typecast on these 4 lines.